chain/node.go

```diff
@@ -52,6 +59,43 @@ func (n *NodeClient) BlockByHash(ctx context.Context, hash string) (*gateway.Blo
 	return b, httpGet(ctx, url, b)
 }
 
+// Account gets account detail by account public key
+func (n *NodeClient) Account(ctx context.Context, pk string) (*gateway.AccountResp, error) {
+	if !isValidPublicKey(pk) {
+		return nil, errPublicKey
+	}
+
+	url := fmt.Sprintf("%s/%s?public_key=%s", n.endpoint, account, pk)
+	a := &gateway.AccountResp{}
+	return a, httpGet(ctx, url, a)
+}
+
+func isValidPublicKey(pk string) bool {
```

Comment:
this is over complicated.
1. strings.ToLower
2. strings.HasPrefix("0x")
3. hex.DecodeString